jraft-extension/rpc-grpc-impl/src/main/java/com/alipay/sofa/jraft/rpc/impl/GrpcServer.java

```diff
@@ -46,26 +60,48 @@
  */
 public class GrpcServer implements RpcServer {
 
-    private final Server                   server;
-    private final MutableHandlerRegistry   handlerRegistry;
-    private final Map<String, Message>     parserClasses;
-    private final MarshallerRegistry       marshallerRegistry;
-    private final RemoteAddressInterceptor remoteAddressInterceptor = new RemoteAddressInterceptor();
-    private final AtomicBoolean            started                  = new AtomicBoolean(false);
+    private static final Logger                       LOG                  = LoggerFactory.getLogger(GrpcServer.class);
+
+    private final Server                              server;
+    private final MutableHandlerRegistry              handlerRegistry;
+    private final Map<String, Message>                parserClasses;
+    private final MarshallerRegistry                  marshallerRegistry;
+    private final List<ServerInterceptor>             serverInterceptors   = new CopyOnWriteArrayList<>();
+    private final List<ConnectionClosedEventListener> closedEventListeners = new CopyOnWriteArrayList<>();
+    private final AtomicBoolean                       started              = new AtomicBoolean(false);
+
+    private ExecutorService                           defaultExecutor;
 
     public GrpcServer(Server server, MutableHandlerRegistry handlerRegistry, Map<String, Message> parserClasses,
                       MarshallerRegistry marshallerRegistry) {
         this.server = server;
         this.handlerRegistry = handlerRegistry;
         this.parserClasses = parserClasses;
         this.marshallerRegistry = marshallerRegistry;
+        registerDefaultServerInterceptor();
     }
 
     @Override
     public boolean init(final Void opts) {
         if (!this.started.compareAndSet(false, true)) {
             throw new IllegalStateException("grpc server has started");
         }
+
+        this.defaultExecutor = ThreadPoolUtil.newBuilder() //
+            .poolName("grpc-default-executor") //
```

Comment:
String variabler "grpc-default-executor" comes out in serveral places, you can define a private final string variable.